From b688a31031768fd6b4b701222b80c98c920dc9da Mon Sep 17 00:00:00 2001
From: Michael Schwartz <mschwart@codeaurora.org>
Date: Tue, 12 Feb 2013 17:50:35 -0800
Subject: [PATCH] Revert "msm7627a: Change adj level and min_free_kbytes"

This reverts commit 9fef2016a3af53f704f2f5ac2841de7e2928266c.  This
change allows the /sys/module/lowmemorykiller/parameters adj and minfree
values to be set by Gecko.

Change-Id: I9f5f3df63089b063d65f514913062957103f9f34
---
 rootdir/etc/init.qcom.post_boot.sh |    8 --------
 1 files changed, 0 insertions(+), 8 deletions(-)

diff --git a/rootdir/etc/init.qcom.post_boot.sh b/rootdir/etc/init.qcom.post_boot.sh
index e0d33bb..b40d2b7 100755
--- a/rootdir/etc/init.qcom.post_boot.sh
+++ b/rootdir/etc/init.qcom.post_boot.sh
@@ -221,11 +221,3 @@ case "$target" in
     ;;
 esac
 
-# Change adj level and min_free_kbytes setting for lowmemory killer to kick in
-case "$target" in
-     "msm7627a")
-	echo 0,1,2,4,9,12 > /sys/module/lowmemorykiller/parameters/adj
-	echo 5120 > /proc/sys/vm/min_free_kbytes
-     ;;
-esac
-
-- 
1.7.8.3

